### ISSUE TITLE
Consider which audio cues for chat experience should be enabled by default

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#185691]
cc @rperez030, @jooyoungseo, @Afre100

Now that we have audio cues for the chat experience, all of which are off by default, we should think about what the best experience would be for most SR users and proceed with that once the final audio cues have been provided.

**Comment by rperez030** [comment]
My choice here would be to only enable audioCues.chatResponsePending  since it is the only one that provides information that is not redundant, however, this is an area in which I believe user research would be beneficial. for example, do people feel like these audio cues complement the screen reader feedback? are they distracting? Wen the chat is done generating the answer, do they prefer to receive screen reader feedback, the audio cue or both?

**Comment by meganrogge** [comment]
Yeah, that was my feeling as well @rperez030. 

Having all off by default would decrease discoverability, so I'm in favor of setting just that one to true

**Comment by AFre100** [comment]
@meganrogge  @rperez030  Just wanted rule out any ambiguity here, does "audioCues.chatResponsePending" = the "progress" or "response" sound?

**Comment by meganrogge** [comment]
I believe he means progress
